lib/prompts/accountNamePrompt.ts

```diff
@@ -12,7 +12,7 @@ type AccountNamePromptResponse = {
 };
 
 export function getCliAccountNamePromptConfig(
-  defaultName?: string
+  defaultName?: string | null
```

Comment:
Would you still need this change if you defaulted to `undefined` [here](https://github.com/HubSpot/hubspot-cli/pull/1365/files#diff-f0f185d80a68501d80d813e663386e095d242d8b6bc4836d250897471ef14735R48)?

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1365/commits/2acdfb8eb4d268b18e4e76619c9101f2aa19d8bc